src/content_script.js

```diff
@@ -40,6 +40,9 @@ const LOGIN_PATTERN_DETECTION_SELECTORS = [
   "[aria-label*='Log in with Facebook']", // Tinder
   "[action*='facebook_login']", // Airbnb
   "[action*='facebook_signup']", // Airbnb
+  "[data-tag='login-form'] ~ span button[color='facebookBlue']", // Patreon
+  ".welcome-buttons .btn-facebook", // Mixcloud homepage
```

Comment:
Can you include `.homepage-photo-leader` too? Also add `div` to the `.btn-facebook` class. _(Similiar to the next comment)_